lib/src/scenario/TestUserManager.ts

```
@@ -0,0 +1,137 @@
+
+  public static users: TestUserModels;
+
+  public static async populateUserModelMap() {
+
+    this.userModelMapEmail = new Map<string, UserModel>();
+    this.userModelMapType = new Map<string, UserModel>();
+
+    for (const user of Object.keys(TestUser)) {
+      const userValue = TestUser[user as keyof typeof TestUser];
+      // Create a user model for each test user
+      const email = this.buildIdentifier(userValue);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)